tests/themes/theme.scss

```diff
@@ -0,0 +1,93 @@
+//------------------------------------------------------------------------------
+// @render: Base-L
```

Comment:
as long as when you get the contents of test you strip all from the folder list of themes